rdrf/registry/patients/models.py

```diff
@@ -404,7 +404,8 @@ class Meta:
         )
 
     def as_dto(self):
-        return PatientDTO(**{f: getattr(self, f) for f in PatientDTO._fields})
+        return PatientDTO(**{**{f: getattr(self, f) for f in patientdto_attr_fields},
+                             **{'working_groups': [wg.name for wg in self.working_groups.all()]}})
```

Comment:
Maybe more efficient to use `self.working_groups.values_list("name", flat=True)`

Reply:
True. I'll update it